lib/loader/index.js

```diff
@@ -420,4 +412,9 @@ function demangle(exports, extendedExports = {}) {
   return extendedExports;
 }
 
-exports.demangle = demangle;
+export default {
+  instantiate,
+  instantiateSync,
+  instantiateStreaming,
+  demangle
+};
```

Comment:
Hmm but all this methods already marked as exported. Is not this redundant?

Reply:
The `default` export is what becomes the transpiled exports of the CommonJS module, i.e.

```js
const loader = require("loader")
// ===
import loader from "loader"; // default export
```

Reply:
I see, thanks!